tmt/base.py

```diff
@@ -1311,6 +1311,7 @@ class Result(object):
         result ........... test execution result
         log .............. one or more log files
         note ............. additional result details
+        duration.......... section stating how long did the test run
```

Comment:
I'd omit `section stating` it is kind of obvious and makes the text too long

Reply:
I changed it to `test execution time (hh:mm:ss)`